docs/docs/userguide/configuration/BITBUCKET_CONFIGURATION.md

```diff
@@ -50,6 +49,13 @@ bitbucket_admin_email_address = "<EMAIL ADDRESS>"
 
     Please refer to [Sensitive Data](#sensitive-data) section.
 
+### Display Name
+Set the display name of the Bitbucket instance. Note that this value is only used during installation and changing the value during an upgrade has no effect.
+
+```terraform
+bitbucket_display_name = "<DISPLAY NAME>"
```

Comment:
I think the current pattern is to use `_`  to delineate instead of space.

Reply:
yea.. Seems like we have inconsistent patterns across the doc. [Example 1](https://github.com/atlassian-labs/data-center-terraform/blob/main/docs/docs/userguide/configuration/CONFIGURATION.md#domain), [example 2](https://github.com/atlassian-labs/data-center-terraform/blob/main/docs/docs/userguide/configuration/BITBUCKET_CONFIGURATION.md#database-instance-class)
For this specific one, I kinda like the space delimiter.. since the value can actually have spaces in between, e.g. `Yifei's Stage Instance`.